public/jsx/views/card-details.jsx

```diff
@@ -57,7 +57,7 @@ module.exports = React.createClass({
         </div>
       );
     } else {
-      return <Spinner text={gettext('Setting up')}/>;
```

Comment:
The one in `getDefaultProps` is just a default which can be overridden by the component rendering Spinner as a prop.